### PR TITLE
Add tablet specific code/Assorted refactors

### PR DESCRIPTION
**Pull request by noahmalmed**
# Description

There are a few things going on here:

1) Dynamicaly Size Text for Android
	- Add `react-native-device-info` this package lets us check if the device is a tabler
	- Add Dynamic sizing for text in the classifier and tutorial

2) Remove drawing modal for tablet
	- We removed the modal because we don't need to create additional space on tablet.

3) Various layout and formatting fixes
	- Fixed an issue with tutorial margins
	- Fixed a bug where the field guide always show on the drawing task

# Review checklist

- [x] Does it work in Android and iOS?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Are tests passing?

### CODE REVIEW

File: src/components/Markings/DrawingClassifier.js [hunk 1]
```diff
@@ -141,6 +145,8 @@ class DrawingClassifier extends Component {
             return <OverlaySpinner overrideVisibility={this.props.isFetching} />
         }
 
+        // We validate that tools has at least one element earlier
+        const tool = this.props.tools[0]
```
Comment (wgranger) [inline src/components/Markings/DrawingClassifier.js]: Is there a chance this could be null? (if used as validation, as the comment says) Also, `this.props.tools[0]` is still used in the line below, as opposed to this new variable.

File: src/components/Markings/DrawingClassifierSubject.js [hunk 2]
```diff
@@ -113,36 +90,24 @@ class ImageWithSvgOverlay extends Component {
     }
 
     render() {
-        const pathPrefix = Platform.OS === 'android' ? 'file://' : ''
```
Comment (wgranger) [inline src/components/Markings/DrawingClassifierSubject.js]: Where has this, and other items for this component, been moved to? It seems this component has changed significantly.

File: src/components/Markings/components/DrawingButtons.js [hunk 3]
```diff
@@ -12,25 +12,25 @@ const DrawingButtons = (props) => {
                 disabled={!props.canUndo}
                 type="undo"
                 activated={false}
-                onPress={() => props.onButtonSelected('undo')}
+                onPress={() => props.onUndoButtonSelected()}
```
Comment (wgranger) [inline src/components/Markings/components/DrawingButtons.js]: Should 'undo' be removed here, or is this now the default action?

File: src/components/Markings/DrawingClassifier.js (hunk 1 above)
Comment (noahmalmed) [reply src/components/Markings/DrawingClassifier.js]: The comment is supposed to point out that tools has been validated to have `length > 0` when we validate the workflow after loading from the API

File: src/components/Markings/DrawingClassifierSubject.js (hunk 2 above)
Comment (noahmalmed) [reply src/components/Markings/DrawingClassifierSubject.js]: So before this refactor, there were two components that essentially did the same thing: `ImageWithSvgOverlay` and `MarkableImage`. They both rendered SVGs over an image, but `MarkableImage` also allowed the user to draw SVGs over the image as well.

As a part of this refactor, I changed `ImageWithSvgOverlay` to `DrawingClassifierSubject`, who's purpose is to display the `DrawingToolView`: which contains the subject and all drawing and SVG logic. And display the blurview and already seen banner over the subject.

In addition, I modified `MarkableImage` to be drawing enabled or disabled, so it can essentially function exactly the same as `ImageWithSvgOverlay` used to function.

So, in short, the functionality wasn't really moved anywhere. It already existed in `MarkableImage` and I'm just leveraging that now

File: src/components/Markings/components/DrawingButtons.js (hunk 3 above)
Comment (wgranger) [reply src/components/Markings/components/DrawingButtons.js]: Ahh yes, my bad. I didn't initially notice the function changed, only the argument.

**Comment by noahmalmed** [comment]
This is ready for re-review

**Comment by wgranger** [comment]
Nice. Looks good on iOS and Android.
